lang/ruby-devel/Portfile

```diff
@@ -33,20 +33,20 @@ long_description    Ruby is the interpreted scripting language \
 homepage            https://www.ruby-lang.org/
 license             {Ruby BSD}
 
-checksums           rmd160  00b4e784882f59c1371e4c27d334fb582b5fcb9c \
-                    sha256  dffcf97eb44e5be51bd1f95cef7d7a82b902667e0fe1f1dcd796c4162eacf968 \
-                    size    15510869
+checksums           rmd160  ea3147611cfba3646baf7f973ca17cbe2561c358 \
+                    sha256  eb4821d72f65293c4b45cd1211d2ad03495207714aa37dacd55109e092b1b39a \
+                    size    15610783
 
 universal_variant   no
 
-# ruby/openssl does not support openssl-3
-openssl.branch      1.1
+# ruby/openssl since ruby-3.2 supports openssl-3
+openssl.branch      3
```

Comment:
This is the default, so I don't think it should be necessary.

https://github.com/macports/macports-ports/blob/3790fe3d680e9b36062b007ef8f507b4595f2c46/_resources/port1.0/group/openssl-1.0.tcl#L32-L36

If for whatever reason it is required, shouldn't the comment be referring to ruby-3.3? Otherwise LGTM. Thanks for the PR @barracuda156 👍

Reply:
Ruby before 3.2 did not build with OpenSSL 3. Since 3.2 it builds. This note is there to register that fact. Otherwise someone gonna wonder if we forgot to put the setting here :)

Reply:
@harens Bump :)